src/net_processing.cpp

```diff
@@ -2535,6 +2535,35 @@ void PeerManager::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDat
         return;
     }
 
+    if (msg_type == NetMsgType::SENDHEADERS) {
```

Comment:
if you move this hunk before the wtxidrelay msg handling, it will avoid a merge conflict with other open pulls

Reply:
Done